config/browserlists.js

```diff
@@ -1,3 +1,4 @@
+// The list of supported browsers
```

Comment:
I'd tweak this to say `The list of all supported browsers`

Reply:
Do we really need this though?
A variable named `fullName` does not need a comment above it that says `The full name` right?